lightwood/api/json_ai.py

```diff
@@ -177,8 +178,7 @@ def generate_json_ai(
 
         }]
 
-        if not problem_definition.timeseries_settings.is_timeseries or \
-                problem_definition.timeseries_settings.nr_predictions <= 1:
+        if not tss.is_timeseries or tss.nr_predictions <= 1:
```

Comment:
when will ``tss.nr_predictions`` ever be less than 1? Might as well do == or 2?

Reply:
Good point, will go with `==`